src/ProjectTemplates/GeneratedContent.targets

```
@@ -40,6 +43,7 @@
       <TemplatePackageVersion_AzureSearchDocuments>11.6.1</TemplatePackageVersion_AzureSearchDocuments>
       <TemplatePackageVersion_CommunityToolkitAspire>9.4.1-beta.291</TemplatePackageVersion_CommunityToolkitAspire>
       <TemplatePackageVersion_MicrosoftExtensionsHosting>10.0.0-preview.6.25358.103</TemplatePackageVersion_MicrosoftExtensionsHosting>
+      <TemplatePackageVersion_MicrosoftExtensionsHttpResilience>9.9.0</TemplatePackageVersion_MicrosoftExtensionsHttpResilience>
```

```suggestion
      <TemplatePackageVersion_MicrosoftExtensionsHttpResilience>9.10.0</TemplatePackageVersion_MicrosoftExtensionsHttpResilience>
```
https://www.nuget.org/packages/Microsoft.Extensions.Http.Resilience/9.10.0?